test/data-source.test.js

```diff
@@ -60,8 +60,8 @@ describe('DataSource', function() {
       });
 
       var Color = ds.createModel('color', {name: String});
-      assert(Color.prototype instanceof loopback.Model);
-      assert.equal(Color.base, loopback.Model);
+      assert(Color.prototype instanceof Color.registry.getModel('Model'));
+      assert.equal(Color.base.modelName, 'PersistedModel');
```

Comment:
I think the answer is that `Color.registry.Model` is not defined, therefore we have to use `getModel()`. I am wondering if it makes sense to expose at least `Model` and `BaseModel` on the `registry` object?